frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss

```diff
@@ -45,6 +45,7 @@
   height: 100%;
   margin-top: 83px;
   overflow-y: scroll;
+  scrollbar-width: none;
```

Comment:
do we need this?

Reply:
we might not, if there won't be more than n amount of meditation topics, that fit into the screen, including rotated screen on mobile?

Reply:
I didn't get your question, but we need to show scrollbar

Reply:
Ok, changed it!